libff/algebra/curves/edwards/edwards_init.cpp

```diff
@@ -176,8 +175,9 @@ void init_edwards_params()
 
     /* pairing parameters */
 
+    edwards_ate_loop_count = bigint_q("4492509698523932320491110403");
+    edwards_final_exponent_last_chunk_abs_of_w0 = bigint_q("17970038794095729281964441603");
```

Comment:
These lines break the CI (`bigint_q` is not in the scope + they are duplicate of the following lines). This is fixed in #90